lib/mindwendel_web/controllers/static_page_html/home.html.heex

```diff
@@ -13,19 +13,20 @@
 
     <div class="row">
       <div class="col-md-12 col-lg-5 mt-2">
-        <%= form_for @changeset, ~p"/brainstormings", fn f -> %>
+        <.simple_form for={@form} action={~p"/brainstormings"} method="post">
           <div class="input-group">
-            <%= text_input(f, :name,
-              class: "form-control",
-              placeholder: gettext("How might we ..."),
-              autofocus: true
-            ) %>
-            <%= if message = f.errors[:name] do %>
-              <span><%= translate_error(message) %></span>
-            <% end %>
-            <%= submit(gettext("Create!"), class: "btn btn-primary fw-bold") %>
+            <.input
+              field={@form[:name]}
+              placeholder={gettext("How might we ...")}
+              form_group_wrapper={false}
```

Comment:
this option took me way too long to find I was diving into the Phoenix code trying to figure out how I can remove it, but it was in our components all along